dropwizard-jersey/src/main/java/io/dropwizard/jersey/DropwizardResourceConfig.java

```diff
@@ -148,6 +149,7 @@ public ResourceConfig register(final Object component) {
                 // Need to create a new subclass dynamically here because Jersey
                 // doesn't add new bindings for the same class
                 ClassPool pool = ClassPool.getDefault();
+                pool.insertClassPath(new LoaderClassPath(this.getClass().getClassLoader()));
```

Comment:
would it be better to make the ClassPool object a member variable instead of creating a new one for each registration?  not sure....